test/module/shared_model/builders/protobuf/transport_builder_test.cpp

```diff
@@ -107,10 +107,9 @@ class TransportBuilderTest : public ::testing::Test {
   //-------------------------------------Block-------------------------------------
   template <typename BlockBuilder>
   auto getBaseBlockBuilder() {
-    return BlockBuilder()
-        .transactions(std::vector<Transaction>({createTransaction()}))
-        .height(1)
-        .createdTime(created_time);
+    std::vector<shared_model::proto::Transaction> txs;
```

Comment:
Here and in other places you could use:
```suggestion
    std::vector<shared_model::proto::Transaction> txs{createTransaction()};
```

Reply:
Not sure if it will work without copy constructor.

Reply:
Andrey is right, this is exactly the thing, which was refactored, because it won't work without a copy constructor. Though it exists now, later it will be removed, so I started the refactoring a little ahead